mythril/analysis/modules/ether_thief.py

```diff
@@ -140,9 +142,13 @@ def _analyze_state(state):
                 gas_used=(state.mstate.min_gas_used, state.mstate.max_gas_used),
             )
         except UnsatError:
-            log.debug("[ETHER_THIEF] no model found")
+            log.debug("No model found")
             return []
 
+        log.debug("Found ether thief issue: %s", issue.as_dict)
+
+        log.debug("Model: %s", solver.pretty_print_model(solver.get_model(constraints)))
```

Comment:
I would probably choose not to print the model in the debug info here, especially if we aren't doing it for the other modules as well.